app/models/organisation.rb

```diff
@@ -146,4 +146,8 @@ def recently_absorbed_organisations_grouped_by_merge_date
 
     absorbed_organisations.merged_during_open_collection_period.group_by(&:merge_date)
   end
+
+  def has_recent_absorbed_organisations?
+    absorbed_organisations&.merged_during_open_collection_period.present?
```

Comment:
do we not want users to see these labels/filters for no longer editable logs?

Reply:
not based on the ticket, I guess it would no longer make sense cause all of the relevant logs would belong to the same organisation and the labels would clutter the page (?)